docker-compose.yml

```diff
@@ -51,6 +51,14 @@ services:
       dockerfile: Dockerfile
     ports:
       - 5055:8080
+    environment:
+        - DAILY_CHECKIN_BASE_URL=https://covid19.dialogue.co/branch/feat-rasa-daily-check-in/#/rasa/ci
```

Comment:
This one is tricky too... should we have a dummy URL? Or assume we use the frontend locally on `localhost:3000`?

Reply:
woops! I forgot to change that one. I will use `localhost:3000/...`